roles/endpoints/defaults/main.yml

```diff
@@ -271,6 +271,7 @@ endpoints:
     name: barbican
     type: key-management
     description: Key Management Service
+    keystone_version: v3
```

Comment:
See other comment, this is logically the keystone_version that the barbican key manager client depends on so it should belong under barbican. I will rename this to `endpoints.barbican.client.keystone_version`